tools/shoestring/tests/commands/test_import_bootstrap.py

```diff
@@ -92,4 +133,26 @@ async def test_cannot_import_when_bootstrap_directory_invalid():
 
 			assert 1 == ex_info.value.code
 
+
+async def test_cannot_import_when_node_key_does_not_exist():
+	# Arrange:
+	with tempfile.TemporaryDirectory() as output_directory:
+		config_filepath = prepare_shoestring_configuration(output_directory, NodeFeatures.PEER)
+
+		with tempfile.TemporaryDirectory() as bootstrap_directory:
+			bootstrap_node_path = Path(bootstrap_directory) / 'nodes/node'
+			bootstrap_node_path.mkdir(parents=True)
+
+			# Act:
+			with pytest.raises(SystemExit) as ex_info:
+				await main([
+					'import-bootstrap',
+					'--config', str(config_filepath),
+					'--bootstrap', str(bootstrap_directory),
+					'--node-key'
```

Comment:
shouldn't you be passing `bootstrap_node_path` ?

Reply:
No, the import-bootstrap requires the root path for the bootstrap.  I will add a comment to clarify.